src/editor/map/context_manager.cpp

```diff
@@ -255,7 +255,10 @@ void context_manager::change_addon_id()
 	std::string new_addon_id = current_addon_;
 	gui2::dialogs::prompt::execute(new_addon_id);
 
-	if(addon_filename_legal(new_addon_id) && filesystem::rename_dir(filesystem::get_current_editor_dir(current_addon_), filesystem::get_current_editor_dir(new_addon_id))) {
+	if(addon_filename_legal(new_addon_id)
```

Comment:
Why was this line broken up?

Reply:
Makes it easier to understand parts of the condition?

Reply:
Can we at least get another opinion on this? I don't like breaking up long lines, personally, but if the general consensus of the project is that it's good, then I'll back off.

Reply:
Result of running `clang-format` on the code:
```diff
258,259c259,262
< 	if(addon_filename_legal(new_addon_id) && filesystem::rename_dir(filesystem::get_current_editor_dir(current_addon_), filesystem::get_current_editor_dir(new_addon_id))) {
< 		std::string main_cfg = filesystem::get_current_editor_dir(new_addon_id)+"/_main.cfg";
---
> 	if(addon_filename_legal(new_addon_id)
> 		&& filesystem::rename_dir(
> 			filesystem::get_current_editor_dir(current_addon_), filesystem::get_current_editor_dir(new_addon_id))) {
> 		std::string main_cfg = filesystem::get_current_editor_dir(new_addon_id) + "/_main.cfg";
```

Reply:
```
std::string old_dir = filesystem::get_current_editor_dir(current_addon_);
std::string new_dir = filesystem::get_current_editor_dir(new_addon_id);
if(addon_filename_legal(new_addon_id) && filesystem::rename_dir(old_dir, new_dir)) {
...
```

Reply:
> Result of running `clang-format` on the code:

This seems even worse than what you did.

Reply:
I agree with Pentarctagon. I'd much rather write the code so clang-format's result is readable instead of discussing exceptions.

The goal should be automatic formatting. This file is a good indicator of how bad we are with consistent formatting otherwise. If there are cases where automatic formatting is unacceptable then that is likely a good indicator that the code is convoluted and should be rewritten. In the absolute worst case formatting can be turned off for a region.

Reply:
Used Pentarctagon's solution to rewrite the check.

Reply:
@CelticMinstrel please resolve this thread if the changes are acceptable.